.gitignore

```diff
@@ -6,7 +6,13 @@ node_modules
 
 # resources created during test run
 .nyc_output
+test/resources/incorrect-site-template/incorrect-site-template-1.0.0.zip
+test/resources/incorrect-site-template/incorrect-site-template-latest.zip
+test/resources/incorrect-site-template/site-template
+test/resources/incorrect-site-template/site/target
+test/resources/incorrect-site-template/theme/package-lock.json
+test/resources/some-site-template/some-site-template-1.0.0.zip
```

Comment:
Can't we just add `test/resources/incorrect-site-template` and `test/resources/some-site-template` instead of all these details?

Reply:
Generally we need to have the test site template in the repo so we can test it / against it. We want to omit only some folders that are getting generated automatically during the build (like the generated .zip files based on the test Site Template). The only thing we can do is to extract test related .gitignore entries into separate .gitignore file which would be placed in the test folder so it does not mix with the regular .gitignore rules.